Makefile

```diff
@@ -7,6 +7,7 @@ GOLANGVERSION = $(shell go version 2>/dev/null | grep -Eo '(go[0-9].[0-9])')
 GOTEST_OPTS = -test.v -check.v
 
 all: $(SUBDIRS)
+	-tests/00-fmt.sh
```

Comment:
We should run `gofmt` before we compile. Otherwise we have to wait for the compile to be complete every time.